test/test-helpers.ts

```diff
@@ -79,3 +80,39 @@ export async function deployERC1155(deployer: any, initialSupply: number) {
   await erc1155Collection.deployed();
   return erc1155Collection;
 }
+
+/**
+ * Constructs a single asset exit pinned to the given chainID and assetHolder address.
+ * The asset is the native asset of the chain.
+ *
+ * @param chainId The qualified asset's chain ID
+ * @param assetHolder the qualified asset's asset holder contract address
+ * @param destination the recipient of the asset
+ * @param amount the amount of the asset to transfer
+ * @returns
+ */
+export function getQualifiedSAE(
+  chainId: number,
+  assetHolder: string,
+  destination: string,
+  amount: string
+): SingleAssetExit {
+  return {
+    asset: "0x0000000000000000000000000000000000000000",
+    assetMetadata: {
+      assetType: AssetType.Qualified,
+      metadata: defaultAbiCoder.encode(
+        ["uint chainID", "address assetHolder"],
+        [chainId, assetHolder]
+      ),
+    },
+    allocations: [
+      {
+        destination: "0x000000000000000000000000" + destination.slice(2),
```

Comment:
This padding is a bit unexpected? Is the passed parameter being treated as an Ethereum address? If so, best to rename that param.

Reply:
376cb74